_topic_map.yml

```diff
@@ -866,6 +866,9 @@ Topics:
 - Name: Pruning objects to reclaim resources
   File: pruning-objects
   Distros: openshift-origin,openshift-enterprise,openshift-webscale
+- Name: The Red Hat Marketplace
```

Comment:
Probably should name it something like "Using the Red Hat Marketplace", to be consistent with our other heading namings.